.github/workflows/Publisher.yml

```
@@ -0,0 +1,65 @@
+      - name: Setup Java (设置 Java 环境)
+        uses: actions/setup-java@v2
+        with:
+          distribution: "temurin"
+          java-version: 17
+
+      - name: Make Gradle Wrapper Executable (使 Gradle Wrapper 可执行)
+        if: ${{ runner.os != 'Windows' }}
+        run: chmod +x ./gradlew
+
+      - name: Read Version from gradle.properties (从 gradle.properties 文件读取版本)
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:2:28: Double quote to prevent globbing and word splitting

(shellcheck)